misc/gaussian/python/gaussianutil.py

```diff
@@ -0,0 +1,173 @@
+import inviwopy as ivw
+import ivwdataframe as df
+import atomdata
+
+import functools
```

Comment:
remove unused imports: functools, itertools and math

Reply:
yep, I have removed them now